packages/button/README.md

```diff
@@ -80,11 +80,26 @@ A block-level button uses 100% of the available width of the container or parent
 <Button block>Submit</Button>
 ```
 
-### ButtonLink
+### Icons
 
-For situations where you need something that has the visual weight of a Button, but the functionality of a link, you can use ButtonLink!
+The `iconAfter` and `iconBefore` props can be used to add system icons to buttons.
 
-ButtonLink adopts the Link component from your chosen router framework, which you can set in the Core component.
+```jsx live
+<ButtonLink
+	iconAfter={ExternalLinkIcon}
+	href="https://steelthreads.github.io/agds-next"
+	target="_blank"
+	rel="noopener noreferrer"
+>
+	Open external link
+</ButtonLink>
+```
+
+## Buttons links
```

Comment:
With the way this page is structured, it makes more sense to have this is as a H2